masonry/src/contexts.rs

```diff
@@ -577,9 +583,32 @@ impl_context_method!(
 pub struct TimerToken;
 
 impl EventCtx<'_> {
-    // TODO - Document
+    // TODO - clearly document all semantics of pointer capture when they've been decided on
     // TODO - Figure out cases where widget should be notified of pointer capture
     // loss
+    /// Capture the pointer in the current widget.
+    ///
+    /// Pointer capture is only allowed during a [`PointerDown`] event. It is a logic error to
+    /// capture the pointer during any other event.
+    ///
+    /// A widget normally only receives pointer events when the pointer is inside the widget's
+    /// layout box. When the pointer is captured by a widget, that widget will continue receiving
+    /// pointer events when the pointer is outside the widget's layout box.
```

Comment:
```suggestion
    /// pointer events when the pointer is outside the widget's layout box (and other widgets will not receive these events).
```

Reply:
Done.

Reply:
In addition to this, I've added context about events bubbling to their parents. 

https://github.com/linebender/xilem/blob/09bc929e021734a564980f4e0622dce66a2229ea/masonry/src/contexts.rs#L592-L597